ui/src/main.js

```diff
@@ -38,20 +38,22 @@ Vue.mixin(NethServer)
 Vue.mixin(UtilService)
 
 // configure i18n
-var langConf = languages.initLang();
-const i18n = new VueI18n({
-  locale: langConf.locale,
-  messages: langConf.messages
-});
-var moment = require("moment");
-moment.locale(langConf.locale);
+const i18n = new VueI18n();
 
 // init Vue app
 var ns = new Vue({
-  el: "#app",
```

Comment:
the `el` attribute actually runs `mount()` immediately: must be removed because language.json is still not loaded